asv.ci.conf.json

```diff
@@ -0,0 +1,21 @@
+ {
+   "version": 1,
+   "project": "astropy",
+   "project_url": "http://www.astropy.org/",
+   "repo": ".",
+   "build_command": [
+     "python -m build --wheel -o {build_cache_dir} {build_dir}"
+   ],
+   "branches": ["main"],
+   "show_commit_url": "http://github.com/astropy/astropy/commit/",
+   "pythons": ["3.11"],
+   "matrix": {
```

Comment:
Do we need to pin dependency versions here to make sure we're comparing apple to apple?

Reply:
We could but in this case we are just comparing 2 commits in the same envs. The env itself isn't that important.